src/section-solving-one-step-equations-involving-percentages.mbx

```diff
@@ -217,18 +331,67 @@
 
         <example>
             <statement>
-                <p>Your monthly internet bill increased from <m>\$40.00</m> to <m>\$48.00</m>. What was the percent of increase?</p>
+                <p>The following shart keeps track of a stock's price change over a week. What was the stock price's percent of increase from Monday to Friday?</p>
```

Comment:
Our students don't have stock portfolios, but they do have monthly bills, which is why the previous version was about a monthly bill. 

This new problem is here so that the graph can be here, which is good. And that means you need something that rises and falls. So it's not as if it could be written to be about a monthly bill. But is there any setting that we could use that would be more "real" to our student population?